src/bokeh/core/property/instance.py

```diff
@@ -153,6 +155,22 @@ def __repr__(self) -> str:
         kwargs = ", ".join(f"{key}={val}" for key, val in self._kwargs.items())
         return f"<Instance: {self._model.__qualified_model__}({kwargs})>"
 
+class InstanceDataclass(Property["dataclass"]):
```

Comment:
Not sure about the name and location.

Reply:
Done in [aa0530e](https://github.com/bokeh/bokeh/pull/14677/commits/aa0530e6ca29e85bb00e559c9f41261e1c57c818)